WalletWasabi.Gui/Controls/NoparaPasswordBox.cs

```diff
@@ -125,7 +126,15 @@ public NoparaPasswordBox()
 
 			try
 			{
-				FontFamily = FontFamily.Parse(fontName); // Use the font.
+				FontFamily ff = FontFamily.SystemFontFamilies.FirstOrDefault(x => x.Name == fontName);
+				if (ff != default)
```

Comment:
I will test this on multiplatform. If it is OK we can use this. Cleaner than this: 
https://stackoverflow.com/a/17596784/1844347

Reply:
I think this api was added to avalonia specifically for this reason... hope it works :)

Reply:
Tested on every OP system it is fine. 
P.S.
The worst thing can happen here is that old issue when the [] rectangles come back.